ui-manchette/src/components/Manchette.tsx

```diff
@@ -59,7 +64,7 @@ const Manchette: FC<ManchetteProps> = ({
     yOffset: 0,
     panning: null,
     scrollPosition: 0,
-    isProportional: false,
+    isProportional: true,
```

Comment:
We want to be by default in proportional ? Or only the story should be like that ?

Reply:
We want it by default in proportional (asked by Loïc)